.github/workflows/master.yml

```diff
@@ -0,0 +1,145 @@
+name: master
+
+on:
+  push:
+    branches: 
+      - master
+  pull_request:
+    branches: 
+      - master
+
+jobs:
+  python3_5:
+    runs-on: ubuntu-latest
+    env:
+      SPARK_VERSION: 2.3.4
+      PANDAS_VERSION: 0.23.4
+      PYARROW_VERSION: 0.10.0
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+
+    - name: setup_python
```

Comment:
Can you name it pretty? e.g.) `Set up Python 3.5`

Reply:
ah, yes. i'll take a look for other also 👍